### PR TITLE
Sort subcategories by name

### PR DESCRIPTION
**Pull request by khrizzcristian**
#### What problem is this solving?
We have added the possibility to sort subcategories by name.

[Workspace with app linked & sort active](https://menu--biasicom.myvtex.com/)
[Default workspace](https://biasicom.myvtex.com/)

#### Screenshots or example usage:

Sort PrintScreen

### CODE REVIEW

**Review by icazevedo — APPROVED** [review]
LGTM

**Review by victorhmp — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Add `sortSubcategories` as option for `category-menu`
```
Comment (icazevedo) [inline CHANGELOG.md:9]:
```suggestion
### Added
- `sortSubcategories` as option for `category-menu`
```

**Review by icazevedo — CHANGES_REQUESTED** [review]
Hey @khrizzcristian! The danger CI is failing because of a little issue with the changelog. Can you please accept the suggestion? We need this before releasing.

I'm unable to fix it directly since you opened the PR from your master branch. If you had done using a different branch, I would be able to do it.

**Review by icazevedo — APPROVED** [review]
I was actually able to do it! Great 😄

**Comment by khrizzcristian** [comment]
Great. Thank you😁

**Review by camilavcoutinho — CHANGES_REQUESTED** [review]

**Review by camilavcoutinho — APPROVED** [review]

**Comment by icazevedo** [comment]
@allcontributors add @khrizzcristian for code

**Pull request merged**